--- a/drivers/mmc/host/sdhci.c
+++ b/drivers/mmc/host/sdhci.c
@@ -3978,14 +3978,14 @@ int __sdhci_add_host(struct sdhci_host *
 		       mmc_hostname(mmc), host->irq, ret);
 		goto untasklet;
 	}
-
+#if 0
 	ret = sdhci_led_register(host);
 	if (ret) {
 		pr_err("%s: Failed to register LED device: %d\n",
 		       mmc_hostname(mmc), ret);
 		goto unirq;
 	}
-
+#endif
 	mmiowb();
 
 	ret = mmc_add_host(mmc);
